src/Test/Integration/DbHelperAbstract.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
```

Comment:
This is the common library. Do you want to introduce the PHP7 stuff here as well? I didn't add strict types for a reason as I found none in the project and didn't know what else is using this repo.